modules/manually-restoring-cluster-etcd-backup.adoc

```diff
@@ -0,0 +1,299 @@
+* A healthy control plane host to use as the recovery host.
+* SSH access to _all_ control plane hosts.
+* All control plane hosts can be accessed through SSH using the root user. 
+* A backup directory containing both the etcd snapshot and the resources for the static pods from the same backup. The file names in the directory must be in the following formats: `snapshot_<datetimestamp>.db` and `static_kuberesources_<datetimestamp>.tar.gz`.
+
+.Procedure
+
+. Select a control plane host to use as the recovery host. This is the host that you will run the restore operation on.
+
+. Use SSH to connect to each of the control plane nodes, including the recovery host.
++
+The Kubernetes API server becomes inaccessible after the restore process starts, so you cannot access the control plane nodes. For this reason, it is recommended to use a SSH connection for each control plane host you are accessing in a separate terminal.
++
+[IMPORTANT]
+====
+If you do not complete this step, you will not be able to access the control plane hosts to complete the restore procedure, and you will be unable to recover your cluster from this state.
+====
+
+. Copy the etcd backup directory to the recovery control plane host.
++
+This procedure assumes that you copied the `backup` directory containing the etcd snapshot and the resources for the static pods to the `/home/core/` directory of your recovery control plane host.
+
+. Stop the static pods on all the control plane nodes; one host at a time.
+
+.. Move the existing Kubernetes API Server static pod manifest out of the kubelet manifest directory.
++
+[source,terminal]
+----
+$ sudo mv /etc/kubernetes/manifests/kube-apiserver-pod.yaml /tmp
+----
+
+.. Verify that the Kubernetes API Server containers have stopped with the command:
++
+[source,terminal]
+----
+$ sudo crictl ps | grep kube-apiserver | grep -v "operator|gaurd"
+----
++
+The output of this command should be empty. If it is not empty, wait a few minutes and check again.
+
+.. If Kubernetes API Server containers haven't stopped yet, force their termination with the command:
++
+[source,terminal]
+----
+$ sudo crictl stop <container_id>
+----
+
+.. Repeat the same steps for `kube-controller-manager-pod.yaml`, `kube-scheduler-pod.yaml` and **eventually** `etcd-pod.yaml`.
+
+.. On each control plane host, move the current `etcd` data into the `backup` folder:
++
+[source,terminal]
+----
+$ sudo mv /var/lib/etcd/member /home/core/assets/backup/
+----
++
+This data will be useful in case the `etcd` backup restore doesn't work and the `etcd` cluster must be restored to the current state.
+
+. Find the correct etcd parameters for each control plane host.
+
+.. The value for `<ETCD_NAME>` is related to the current host, and it is equal to the value of the `ETCD_NAME` variable in the manifest `/etc/kubernetes/static-pod-resources/etcd-certs/configmaps/restore-etcd-pod/pod.yaml` file in the current host. It can be found with the command:
```

Comment:
```suggestion
.. The value for `<ETCD_NAME>` is unique for the each control plane host, and it is equal to the value of the `ETCD_NAME` variable in the manifest `/etc/kubernetes/static-pod-resources/etcd-certs/configmaps/restore-etcd-pod/pod.yaml` file in the specific control plane host. It can be found with the command:
```

Reply:
Ack.